.github/workflows/ci-vulnerability-scans.yml

```diff
@@ -79,7 +96,15 @@ jobs:
         uses: anchore/scan-action@v3
         with:
           image: ${{ steps.build-image.outputs.image }}
-          output-format: "table"
+          output-format: json
+
+      - name: Save output to workflow summary
+        if: always() # Runs even if there is a failure
+        run: |
+          echo "# Anchore Results" >> $GITHUB_STEP_SUMMARY
+          if [ -f results.json ]; then
+            cat results.json >> $GITHUB_STEP_SUMMARY
+          fi
```

Comment:
given the options of:
- human readable but have to look at logs
- json but can look at summary

I think i'll still choose human readable. so could we still set `output-format` to `table` but then just provide instructions in the summary e.g.


```suggestion
          output-format: table

      - name: Save output to workflow summary
        if: always() # Runs even if there is a failure
        run: echo "Look in GitHub Action logs for Anchore scan results" >> $GITHUB_STEP_SUMMARY
```